src/common/constants.js

```diff
@@ -4,3 +4,8 @@ export const APP_VERSION = packagefile.version
 export const appName = 'translationCore:Create'
 export const base_url = 'https://git.door43.org'
 export const tokenid = 'PlaygroundTesting'
+
+export const MANIFEST_NOT_FOUND_ERROR = 'This resource manifest failed to load.  Please confirm that the correct manifest.yaml file exists in the resource at:\n'
```

Comment:
> This resource manifest

Was this message approved by @birchamp and the QA Team (@elsylambert, @cckozie)?